include/maidsafe/routing/node_info.h

```diff
@@ -34,43 +34,36 @@ namespace routing {
 struct NodeInfo {
   using PublicPmid = passport::PublicPmid;
 
-  NodeInfo(NodeId id, PublicPmid dht_fob) : id(id), dht_fob(std::move(dht_fob)) {}
-  NodeInfo() = default;
+  NodeInfo(NodeId id_in, PublicPmid dht_fob_in, bool connected_in)
+      : id(id_in), dht_fob(std::move(dht_fob_in)), connected(connected_in) {}
+
+  NodeInfo() : id(), dht_fob(), connected(false) {}
```

Comment:
There is still a default c'tor - I just wanted to explicitly initialise the `bool` member to `false` rather than a random value.

Also, I removed the Cereal functions from this class since they're not needed, so we can perhaps remove the default c'tor altogether.